documentation/4.0/content/base-images/custom-images.md

```diff
@@ -506,10 +506,103 @@ to create the domain home in Domain in Image.
 
 #### Create a custom image with your model inside the image
 
-In the [Model in Image]({{< relref "/managing-domains/model-in-image/_index.md" >}})
-documentation, you will see a reference to a "base" or `--fromImage` image. You should use an image with
-the recommended security patches installed as this base image, where this image could be an OCR image or a custom image.
+{{% notice tip %}}
+This section describes an option for layering Model in Image model files on an WebLogic image.
+The preferred approach for supplying Model in Image files
+is to instead use [Auxiliary images]({{< relref "/managing-domains/model-in-image/auxiliary-images.md" >}}).
+{{% /notice %}}
 
-See [Obtain images from the Oracle Container Registry]({{< relref "/base-images/ocr-images#obtain-images-from-the-oracle-container-registry" >}})
-or
-[Create a custom image with patches applied](#create-a-custom-image-with-patches-applied).
+{{% notice warning %}}
+The example in this section references base image
+`container-registry.oracle.com/middleware/weblogic:12.2.1.4`.
+This is an OCR General Availability (GA) image
+which **does not include** the latest security patches for WebLogic Server.
+GA images are intended for single desktop demonstration and development purposes _only_.
+For all other purposes, Oracle strongly recommends using only images with the latest set of recommended patches applied,
+such as OCR Critical Patch Updates (CPU) images or custom generated images.
+See [Ensure you are using recently patched images]({{< relref "/base-images/ocr-images#ensure-you-are-using-recently-patched-images" >}}).
+{{% /notice %}}
+
+Example steps for creating a custom WebLogic image with a Model in Image file layer
+(using files from the Model in Image sample):
+
+1. Read the [Model in Image User Guide]({{< relref "/managing-domains/model-in-image/_index.md" >}})
+   and the [Model in Image Sample]({{< relref "/managing-domains/samples/domains/model-in-image/_index.md" >}})
+   to gain an overall understanding of Model in Image domains.
+   Note that the sample uses the most recommended approach,
+   auxiliary images, instead of the alternative approach used in this example.
```

Comment:
Not sure what you mean. Can you provide the entire rewritten sentence?

Reply:
Sure. -> Note that the sample uses the most recommended approach, auxiliary images, instead of the alternative approach, which is used in this example. (Ah, I see my mistake. Previously, I left out the word "is". Sorry for the confusion.)

Reply:
Not sure what you mean. I've reworded to `Note that the sample uses the recommended best approach,
   auxiliary images, instead of the alternative approach which is used in this example.`

Reply:
done